app/addons/databases/stores.js

```diff
@@ -112,8 +112,15 @@ const DatabasesStoreConstructor = FauxtonAPI.Store.extend({
     if (!details) {
       return {};
     }
+
+    const sizeReporter = FauxtonAPI.getExtensions('DatabaseList:dbSizeReporter');
```

Comment:
This is a real nitpick but I think it should be `dbSizeReportFieldName` or `dbSizeReporterName`